test/integration/setup.js

```diff
@@ -169,9 +169,10 @@ async function setup() {
     coverNFTDescriptor.address,
   ]);
 
-  // 2. deploy Cover, StakingProducts and TokenController proxies
+  // 2. deploy Cover, StakingProducts, CoverProducts and TokenController proxies
   let cover = await deployProxy('Stub');
   let stakingProducts = await deployProxy('Stub');
+  let coverProducts = await deployProxy('Stub');
```

Comment:
We don't seem to need a stub for this contract, deploy it directly so we won't need an upgrade tx.